docs/guides/tutorials/nextjs_app_router.rst

```diff
@@ -541,6 +541,16 @@ and create a new instance. Create and make note of a secret key for your EdgeDB
 can create a new secret key from the "Secret Keys" tab in the EdgeDB Cloud 
 console. We'll need this later to connect to the database from Vercel.
 
+.. note::
+
+  If you want to restore your data from a local instance to the cloud, you 
+  can use the ``edgedb dump`` and ``edgedb restore`` commands.
+
+.. code-block:: bash
+
+  $ edgedb dump <your-dump.dump>
+  $ edgedb restore -I <org>/<instance-name> <your-dump.dump>
```

Comment:
For whatever we end up recommending below this, we may want to mention here that migrating is no longer necessary if you choose to dump/restore.